src/ledger/fuzz.zig

```diff
@@ -0,0 +1,466 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+const build_options = @import("build-options");
+const ledger = @import("lib.zig");
+
+const ColumnFamily = sig.ledger.database.ColumnFamily;
+
+const allocator = std.heap.c_allocator;
+
+const Data = struct {
+    value: []const u8,
+};
+
+const cf1 = ColumnFamily{
+    .name = "data",
+    .Key = u64,
+    .Value = Data,
+};
+
+var dataMap = std.AutoHashMap(u32, Data).init(allocator);
+var dataKeys = std.ArrayList(u32).init(allocator);
```

Comment:
i dont think we actually need this - we can just use dataMap.keys() when we need to

Reply:
I actually had that but had some issues. So just quickly went with keeping a separate list. I'll revisit this and see if I can drop it

Reply:
> I actually had that but had some issues.

It was the fact that `AutoHashMap` don't have `.keys()` method, but it only returns a keyIterator. So added a separate list to quickly get going

Dropped it in [c695ce7](https://github.com/Syndica/sig/pull/452/commits/c695ce776cf0e94aa2a502233de86c3696196a62)

Reply:
should still be `data_map`